Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Taxonomy of Failsafe Levels
+type: Decision Record
+status: Draft
+track: IaaS
+---
+
+
+## Abstract
+
+When talking about redundancy and backups in the context of cloud infrastructures, the scope under which circumstances these concepts apply to various resources is neither homogenous nor intuitive.
+There does exist very detailed lists of risks and what consequences there are for each risk, but this Decision Record should give a high-level view on the topic.
+So that in each standard that referenced redundancy, it can easily be seen how far this redundancy goes in that certain circumstance.
+Readers of such standards should be able to know at one glance, whether the achieved failure safeness is on a basic level or a higher one and whether there would be additional actions needed to protect the data.
+
+This is why this decision record aims to define different levels of failure safety.
+These levels can then be used in standards to clearly set the scope that certain procedures in e.g. OpenStack offer.
+
+## Glossary
+
+| Term                | Explanation                                                                                                                              |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| Availability Zone   | (also: AZ) internal representation of physical grouping of service hosts, which also lead to internal grouping of resources.             |
+| BSI                 | German Federal Office for Information Security (Bundesamt für Sicherheit in der Informationstechnik).                                    |
+| CSP                 | Cloud Service Provider, provider managing the OpenStack infrastructure.                                                                  |
+| Compute             | A generic name for the IaaS service, that manages virtual machines (e.g. Nova in OpenStack).                                             |
+| Network             | A generic name for the IaaS service, that manages network resources (e.g. Neutron in OpenStack).                                         |
+| Storage             | A generic name for the IaaS service, that manages the storage backends and virtual devices (e.g. Cinder in OpenStack).                   |
+| RTO                 | Recovery Time Objective.                                                                                                                 |
```

Comment:
```suggestion
| RTO                 | Recovery Time Objective, the time within which a service must be restored.                                                               |
```

Reply:
"the acceptable time"

Reply:
I wrote in other words, because to me your sentence was confusing. Please check, if that is, what you wanted to say.

Reply:
yeah, that works for me